apps/auth-admin-web/specs/index.spec.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { render } from '@testing-library/react'
-
+import 'whatwg-fetch'
```

Comment:
Is this accidental?

Reply:
No, seemed like the only way to get fetching to work inside the test, on fetch the test is using relative paths where the default global fetch API introduced in v18 was failing, I tried `node-fetch` too but this was also failing.